tests/gnrc_dhcpv6_client/dhcpv6_server.sh

```diff
@@ -25,26 +25,25 @@ _kea_version_lesser_1_7_10() {
 
 _dhcpv6_server() {
     TMPDIR="$(mktemp -u | xargs dirname)/${APPLICATION}_kea"
+    CONFIG="${TMPDIR}/kea-dhcp6.config"
 
     # only used `mktemp` with dry-run above to get temp directory name, so we
     # still need to create the directory
     mkdir -p "${TMPDIR}"
-    if _kea_version_lesser_1_7_10; then
-        config=$2
-    else
+    sed "s/\"tapbr0\"/\"${IFACE}\"/" "$2" > "${CONFIG}"
```

Comment:
I think think this makes the conf file a template -> i think this should be more visible to the person looking at it

```suggestion
    sed "s/\"--IFACE--\"/\"${IFACE}\"/" "$2" > "${CONFIG}"
```

Reply:
and since python is used for one case maybe it would be god for both and have it in a script instead of inline

Reply:
wrong PR this shoule go to #16796